### PR TITLE
Add comments to dependabot.yml

### PR DESCRIPTION
**Pull request by victorlin**
### Description of proposed changes

Adds some useful links.

### Related issue(s)

_N/A_

### Testing

_N/A_

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,7 +1,11 @@
+# Dependabot configuration file
+# Reference: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
 version: 2
 updates:
   - package-ecosystem: "npm"
     directory: "/"
+    # If you want to run this manually, go to https://github.com/nextstrain/nextstrain.org/network/updates
+    # and click "Last checked …" then "Check for updates".
     schedule:
       interval: "daily"
```
Comment (jameshadfield) [inline .github/dependabot.yml:10]: Can we make this monthly? 🙏

Comment (victorlin) [reply .github/dependabot.yml:10]: @jameshadfield I'm assuming you want the security updates to be monthly, but [that isn't possible](https://github.com/nextstrain/auspice/issues/1462#issuecomment-1270750330). This is just for the Auspice version checks, which should be more useful on a daily interval.

Comment (jameshadfield) [reply .github/dependabot.yml:10]: That's exactly what I wanted. Oh well :( thanks for looking into it!

> This is just for the Auspice version check

Where doe the config specify only auspice? (Asking for my own understanding.)

Comment (victorlin) [reply .github/dependabot.yml:10]: The last line! Not so obvious from the PR changes view.

https://github.com/nextstrain/nextstrain.org/blob/c2f3d84a398dc9cb9de55bf632f63c52d5f47826/.github/dependabot.yml#L5-L12

**Review by jameshadfield — APPROVED** [review]

**Pull request merged**